[2024-06-20T11:35:21.515+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-20T11:35:21.533+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_data_etl_v30.data_etl.preprocess_data.filter_tags scheduled__2024-06-18T00:00:00+00:00 [queued]>
[2024-06-20T11:35:21.539+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_data_etl_v30.data_etl.preprocess_data.filter_tags scheduled__2024-06-18T00:00:00+00:00 [queued]>
[2024-06-20T11:35:21.540+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-06-20T11:35:21.547+0000] {taskinstance.py:2330} INFO - Executing <Task(_PythonDecoratedOperator): data_etl.preprocess_data.filter_tags> on 2024-06-18 00:00:00+00:00
[2024-06-20T11:35:21.552+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_data_etl_v30', 'data_etl.preprocess_data.filter_tags', 'scheduled__2024-06-18T00:00:00+00:00', '--job-id', '133', '--raw', '--subdir', 'DAGS_FOLDER/data_etl_dag.py', '--cfg-path', '/tmp/tmpvd61g_kn']
[2024-06-20T11:35:21.554+0000] {standard_task_runner.py:91} INFO - Job 133: Subtask data_etl.preprocess_data.filter_tags
[2024-06-20T11:35:21.554+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=1058) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-06-20T11:35:21.556+0000] {standard_task_runner.py:63} INFO - Started process 1060 to run task
[2024-06-20T11:35:21.579+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_data_etl_v30.data_etl.preprocess_data.filter_tags scheduled__2024-06-18T00:00:00+00:00 [running]> on host 10482076056f
[2024-06-20T11:35:21.632+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Solarev Ruslan' AIRFLOW_CTX_DAG_ID='dag_data_etl_v30' AIRFLOW_CTX_TASK_ID='data_etl.preprocess_data.filter_tags' AIRFLOW_CTX_EXECUTION_DATE='2024-06-18T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-18T00:00:00+00:00'
[2024-06-20T11:35:21.632+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-20T11:35:21.639+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-20T11:35:21.640+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/decorators/base.py", line 265, in execute
    return_value = super().execute(context)
                   ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/tasks/data_preprocessing/filter_tags.py", line 32, in filter_tags
    dataset = pd.read_csv(preproc_data_path)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/pandas/io/parsers/readers.py", line 948, in read_csv
    return _read(filepath_or_buffer, kwds)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/pandas/io/parsers/readers.py", line 611, in _read
    parser = TextFileReader(filepath_or_buffer, **kwds)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/pandas/io/parsers/readers.py", line 1448, in __init__
    self._engine = self._make_engine(f, self.engine)
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/pandas/io/parsers/readers.py", line 1705, in _make_engine
    self.handles = get_handle(
                   ^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/pandas/io/common.py", line 718, in get_handle
    ioargs = _get_filepath_or_buffer(
             ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/pandas/io/common.py", line 460, in _get_filepath_or_buffer
    raise ValueError(msg)
ValueError: Invalid file path or buffer object type: <class 'NoneType'>
[2024-06-20T11:35:21.647+0000] {taskinstance.py:1206} INFO - Marking task as UP_FOR_RETRY. dag_id=dag_data_etl_v30, task_id=data_etl.preprocess_data.filter_tags, run_id=scheduled__2024-06-18T00:00:00+00:00, execution_date=20240618T000000, start_date=20240620T113521, end_date=20240620T113521
[2024-06-20T11:35:21.654+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 133 for task data_etl.preprocess_data.filter_tags (Invalid file path or buffer object type: <class 'NoneType'>; 1060)
[2024-06-20T11:35:21.689+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-06-20T11:35:21.700+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-20T11:35:21.704+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
